wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -78,9 +76,9 @@ private void setupGooglePhotos() throws Exception {
 
     private void sendToGmail(String recipient) throws Exception {
         String gID = gmail.packageID;
-
+        
         shareUsingApp("Gmail", "gmail");
-
+        
```

Comment:
Yea they are, I've checked the file in my text editor and can also see the spaces there. They've most likely been added automatically by your editor when you re-added the line.